index.html

```diff
@@ -370,10 +367,11 @@ <h2 id="transcode">Change codec (transcode)</h2>
         <dt><i>output_file</i></dt><dd>path and name of the output file</dd>
       </dl>
       <p>A couple notes</p>
-        <ul>
-          <li>About ID3v2.3 tag: ID3v2.3 is better supported than ID3v2.4, FFmpeg's default ID3v2 setting.</li>
-          <li>About dither methods: FFmpeg comes with a variety of dither algorithms, outlined in the <a href="https://ffmpeg.org/ffmpeg-resampler.html" target="_blank">official docs</a>, though some may lead to unintended, drastic digital clipping on some systems.</li>
-        </ul>
+      <ul>
+        <li>About ID3v2.3 tag: ID3v2.3 is better supported than ID3v2.4, FFmpeg's default ID3v2 setting.</li>
+        <li>About dither methods: FFmpeg comes with a variety of dither algorithms, outlined in the <a href="https://ffmpeg.org/ffmpeg-resampler.html" target="_blank">official docs</a>, though some may lead to unintended, drastic digital clipping on some systems.</li>
+      </ul>
```

Comment:
👍 alignment

Reply:
Looking at alignment at the very bottom of the doc right now... agggghhhh! There is an indention off somewhere...

Reply:
Found it, haha.

Reply:
:-)))